src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.vue

```diff
@@ -1,13 +1,10 @@
 <template>
-  <span class="va-count-badge">
-    <slot></slot>
-    <span
-      class="va-count-badge__round"
-      v-if="displayNumber !== '0'"
-      :style="computedStyle"
-    >
-      {{ displayNumber }}
-    </span>
+  <span
+    class="va-count-badge"
+    v-if="displayNumber !== '0'"
```

Comment:
`0` is a legit number to display inside the badge. Show/hide should be handled outside of this component.